sevctl/src/main.rs

```diff
@@ -22,18 +22,18 @@ const VERSION: &str = env!("CARGO_PKG_VERSION");
 const AUTHORS: &str = env!("CARGO_PKG_AUTHORS");
 
 fn download(rsp: reqwest::Result<reqwest::blocking::Response>, usage: Usage) -> sev::Certificate {
-    let mut rsp = rsp.expect(&format!("unable to contact {} server", usage));
+    let mut rsp = rsp.unwrap_or_else(|_| panic!("unable to contact {} server", usage));
 
     if !rsp.status().is_success() {
         panic!("received failure from {} server: {}", usage, rsp.status());
     }
 
     let mut buf = Vec::new();
     rsp.copy_to(&mut buf)
-        .expect(&format!("unable to complete {} download", usage));
+        .unwrap_or_else(|_| panic!("unable to complete {} download", usage));
 
     sev::Certificate::decode(&mut &buf[..], ())
-        .expect(&format!("unable to parse downloaded {}", usage))
+        .unwrap_or_else(|_| panic!("unable to parse downloaded {}", usage))
```

Comment:
Does clippy still complain if we don't borrow the original `format!()` statement? I feel like this is precisely the use-case for `expect()`

Reply:
> Does clippy still complain if we don't borrow the original `format!()` statement? I feel like this is precisely the use-case for `expect()`

```
error[E0308]: mismatched types
  --> src/main.rs:25:30
   |
25 |     let mut rsp = rsp.expect(format!("unable to parse downloaded {}", usage));
   |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `&str`, found struct `std::string::String`
   |
```

Reply:
but thinking more about it, with `unwrap_or_else` we should also output the error. Will fix.